pkg/web/apps/apps_service_test.go

```diff
@@ -52,3 +52,66 @@ func Test_appsService_GetApps(t *testing.T) {
 		})
 	}
 }
+
+func Test_appsService_GetAppsID(t *testing.T) {
+	tests := []struct {
+		name        string
+		id          string
+		want        *[]models.App
+		wantMessage string
+	}{
+		{
+			name: "appsService.GetAppsID() should return a single app and Success,OK",
+			id:   "a986f8d1-aff7-45e0-8fc6-cec2abc0234d",
+			want: &[]models.App{
+				models.App{
+					ID:      "a986f8d1-aff7-45e0-8fc6-cec2abc0234d",
+					AppID:   "com.aerogear.app1",
+					AppName: "app1",
+					DeployedVersions: []models.AppVersion{
+						{
+							ID:               1,
+							Version:          "com.aerogear.app1",
+							AppID:            "A1B2C3D4E5.com.domainname.applicationname",
+							Disabled:         false,
+							DisabledMessage:  "Disabled by admin",
+							NumOfClients:     11,
+							NumOfAppStartups: 12,
+						},
+						{
+							ID:               2,
+							Version:          "com.aerogear.app2",
+							AppID:            "ABCDEFGH12.com.domainname.applicationname",
+							Disabled:         true,
+							DisabledMessage:  "Disabled by admin",
+							NumOfClients:     100,
+							NumOfAppStartups: 100000,
+						},
+					},
+				},
+			},
+			wantMessage: "Success,OK",
+		},
+		{
+			name: "appsService.GetAppsID() should return a App Not Found",
+			id:   "d986f8d1-aff7-45e0-8fc6-cec2abc0234d",
+			want: nil,
+			wantMessage: "App Not Found",
+		},
```

Comment:
I would add one additional unit test for the case where an error is returned from the repository. `want` will be a standard Go error

Reply:
This will have to wait until your changes are merged in , Don't believe I can change the `want` to an error , but with your changes the function will be returning an error rather that a string and I can check that.